addons/mail/static/tests/tours/discuss_public_tour.js

```diff
@@ -37,5 +39,41 @@ tour.register(
             content: "Wait for all modules loaded check in previous step",
             trigger: ".o_mail_channel_public_modules_loaded",
         },
+        {
+            content: "Write something in composer",
+            trigger: ".o-mail-composer-textarea",
+            run: "text cheese",
```

Comment:
Weren't these already added in another PR?
They should be gone at rebase.

Reply:
it's in another file, this is for the user test and the other was for the guest